src/test/java/com/easypost/WebhookTest.java

```diff
@@ -0,0 +1,113 @@
+package com.easypost;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Webhook;
+import com.easypost.model.WebhookCollection;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.MethodOrderer;
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class WebhookTest {
+    private static Map<String, Object> params = new HashMap<>();
+    private static Webhook webhook;
+
+    /**
+     * Setup the testing environment for this file.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @BeforeAll
+    public static void setup() throws EasyPostException {
+        EasyPost.apiKey = System.getenv("EASYPOST_TEST_API_KEY");
+
+        params.put("url", "http://example.com");
+
+        webhook = Webhook.create(params);
+    }
+
+    /**
+     * Test creating a webhook.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @Test
+    @Order(1)
+    public void testCreate() throws EasyPostException {
+        Map<String, Object> p = new HashMap<>();
+        p.put("url", "http://examples.com");
+
+        Webhook webhook = Webhook.create(p);
+
+        assertTrue(webhook instanceof Webhook);
+        assertTrue(webhook.getId().startsWith("hook_"));
+        assertEquals("http://examples.com", webhook.getUrl());
+
+        webhook.delete();
```

Comment:
Please add an inline comment stating we are deleting the webhook here so we don't keep sending events to a dead webhook.

Reply:
Inline comments are "inline" because they either happen right before the line they are referencing or are on the same line. Stating the comment after the code it's annotating can be misleading. Please move the comment up a line.